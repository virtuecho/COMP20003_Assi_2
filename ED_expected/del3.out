COMP20003 Code,Official Code Suburb,Official Name Suburb,Year,Official Code State,Official Name State,Official Code Local Government Area,Official Name Local Government Area,Latitude,Longitude
