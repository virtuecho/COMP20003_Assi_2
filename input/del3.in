Springfield
Richmond
Windsor
Little Back Creek
Carlton River
Docklands
Melbourne
Kingswood
Richmond
Williamstown
Mount Richmond
Greenlands
South Windsor
Back Creek
West Melbourne
Carlton North
Red Hills
Greenlands
Glenroy
Parkville
Richmond Lowlands
Glenroy
Buckingham
Richmond Hill
Windsor
Yarraville
Ascot
Williamstown
Far Far Away
Richmond
Richmond Plains
Windsor
Spotswood
Carlton
Kensington Grove
Glenroy
Williamstown North
Kensington Gardens
Richmond Hill
Greenlands
Springfield Lakes
Kingswood
Seddon
Kingswood
Parkville
Seddon
Melbourne Airport
Windsor
Carlton
Official Name Suburb
Windsor Gardens
Glenroy
West Richmond
Richmond Vale
East Melbourne
Kensington
Cedar Creek
Williamstown
Ascot Vale
Kingswood
Springfield Central
Glenroy
Red Hill
Kensington Park
North Melbourne
Kensington
South Melbourne
Ascot Park
Carlton
Kensington
Kensington
North Richmond
Red Hill South
Kensington
Port Melbourne
Windsor Downs
No More Land
Empty Land
