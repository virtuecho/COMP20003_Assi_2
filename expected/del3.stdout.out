query= Springfield
Springfield --> 3 records deleted
query= Richmond
Richmond --> 6 records deleted
query= Windsor
Windsor --> 4 records deleted
query= Little Back Creek
Little Back Creek --> 1 records deleted
query= Carlton River
Carlton River --> 1 records deleted
query= Docklands
Docklands --> 1 records deleted
query= Melbourne
Melbourne --> 1 records deleted
query= Kingswood
Kingswood --> 4 records deleted
query= Richmond
Richmond --> NOT FOUND
query= Williamstown
Williamstown --> 3 records deleted
query= Mount Richmond
Mount Richmond --> 1 records deleted
query= Greenlands
Greenlands --> 3 records deleted
query= South Windsor
South Windsor --> 1 records deleted
query= Back Creek
Back Creek --> 6 records deleted
query= West Melbourne
West Melbourne --> 1 records deleted
query= Carlton North
Carlton North --> 1 records deleted
query= Red Hills
Red Hills --> 1 records deleted
query= Greenlands
Greenlands --> NOT FOUND
query= Glenroy
Glenroy --> 5 records deleted
query= Parkville
Parkville --> 2 records deleted
query= Richmond Lowlands
Richmond Lowlands --> 1 records deleted
query= Glenroy
Glenroy --> NOT FOUND
query= Buckingham
Buckingham --> 3 records deleted
query= Richmond Hill
Richmond Hill --> 2 records deleted
query= Windsor
Windsor --> NOT FOUND
query= Yarraville
Yarraville --> 1 records deleted
query= Ascot
Ascot --> 5 records deleted
query= Williamstown
Williamstown --> NOT FOUND
query= Far Far Away
Far Far Away --> NOT FOUND
query= Richmond
Richmond --> NOT FOUND
query= Richmond Plains
Richmond Plains --> 1 records deleted
query= Windsor
Windsor --> NOT FOUND
query= Spotswood
Spotswood --> 1 records deleted
query= Carlton
Carlton --> 3 records deleted
query= Kensington Grove
Kensington Grove --> 1 records deleted
query= Glenroy
Glenroy --> NOT FOUND
query= Williamstown North
Williamstown North --> 1 records deleted
query= Kensington Gardens
Kensington Gardens --> 1 records deleted
query= Richmond Hill
Richmond Hill --> NOT FOUND
query= Greenlands
Greenlands --> NOT FOUND
query= Springfield Lakes
Springfield Lakes --> 1 records deleted
query= Kingswood
Kingswood --> NOT FOUND
query= Seddon
Seddon --> 2 records deleted
query= Kingswood
Kingswood --> NOT FOUND
query= Parkville
Parkville --> NOT FOUND
query= Seddon
Seddon --> NOT FOUND
query= Melbourne Airport
Melbourne Airport --> 1 records deleted
query= Windsor
Windsor --> NOT FOUND
query= Carlton
Carlton --> NOT FOUND
query= Official Name Suburb
Official Name Suburb --> NOT FOUND
query= Windsor Gardens
Windsor Gardens --> 1 records deleted
query= Glenroy
Glenroy --> NOT FOUND
query= West Richmond
West Richmond --> 1 records deleted
query= Richmond Vale
Richmond Vale --> 1 records deleted
query= East Melbourne
East Melbourne --> 1 records deleted
query= Kensington
Kensington --> 5 records deleted
query= Cedar Creek
Cedar Creek --> 4 records deleted
query= Williamstown
Williamstown --> NOT FOUND
query= Ascot Vale
Ascot Vale --> 1 records deleted
query= Kingswood
Kingswood --> NOT FOUND
query= Springfield Central
Springfield Central --> 1 records deleted
query= Glenroy
Glenroy --> NOT FOUND
query= Red Hill
Red Hill --> 8 records deleted
query= Kensington Park
Kensington Park --> 1 records deleted
query= North Melbourne
North Melbourne --> 1 records deleted
query= Kensington
Kensington --> NOT FOUND
query= South Melbourne
South Melbourne --> 1 records deleted
query= Ascot Park
Ascot Park --> 1 records deleted
query= Carlton
Carlton --> NOT FOUND
query= Kensington
Kensington --> NOT FOUND
query= Kensington
Kensington --> NOT FOUND
query= North Richmond
North Richmond --> 1 records deleted
query= Red Hill South
Red Hill South --> 1 records deleted
query= Kensington
Kensington --> NOT FOUND
query= Port Melbourne
Port Melbourne --> 1 records deleted
query= Windsor Downs
Windsor Downs --> 1 records deleted
query= No More Land
No More Land --> NOT FOUND
query= Empty Land
Empty Land --> NOT FOUND
Summary: 78 deleted operations performed, 100 records deleted in total
